indeex.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="stylesheet.css">
+    <header>
+        <!-- section which is not needed -->
+    <div class="logo"><!-- logo will be put here --></div>
+    <nav> 
+        <ul>
+            <li><!--Home --></li>
+            <li><!--About --></li>
+            <li><!--Services --></li>
+            <li><!-- Gallery --></li>
+            <li><!--Blogs --></li>
+            <li><!-- Contact--></li>
+
+        </ul>
+    </nav> 
```

Comment:
what is this <nav> tag doing?

Reply:
i have created the navigation here, which will be used for the next task, that's why I commented for now.

Reply:
resolved!